### PR TITLE
Generic popup with pepcus core library.

### PR DESCRIPTION
**Pull request by BhagyashreeBhoyar**
A generic pop-up with pepcus core library.

### CODE REVIEW

File: src/constants/theme.js [hunk 1]
```diff
@@ -47,6 +47,14 @@ const APP_THEME = {
       light: '#dbd7d6',
       text: '#010306',
     },
+    popup: {
+      borderColor: 'rgb(249, 87, 10)',
+      color: '#dbd7d6',
```
Comment (Nikmahajan) [inline src/constants/theme.js]: Use color from a `COLORS` object and define all the colors there, so we can avoid redundancy and maintain consistency.

File: src/components/routeComponents/memberRegistrationForm/SuccessMessagePopup.jsx [hunk 2]
```diff
@@ -43,19 +44,21 @@ const SuccessMessagePopup = ({
...
-        <Button
-          color="tertiary"
-          width="170px"
-          margin="10px 25px"
-          onClick={redirectToPreviousLocation}
-        >
-          {goBackBtnText}
-        </Button>
+        <Row width="100%" justify="center">
+          <Typography type="body" fontSize="16px">{REGISTRATION_SUCCESS_MESSAGE}</Typography>
```
Comment (Nikmahajan) [inline src/components/routeComponents/memberRegistrationForm/SuccessMessagePopup.jsx]: Create a component which returns `<Typography type="body" fontSize="16px">`.

File: src/constants/theme.js (hunk 1 above)
Comment (BhagyashreeBhoyar) [reply src/constants/theme.js]: Done.

File: src/components/routeComponents/memberRegistrationForm/SuccessMessagePopup.jsx (hunk 2 above)
Comment (BhagyashreeBhoyar) [reply src/components/routeComponents/memberRegistrationForm/SuccessMessagePopup.jsx]: Done.

**Pull request merged**